Documentation/ipvlan.md

```diff
@@ -38,3 +38,40 @@ Because all ipvlan interfaces share the MAC address with the host interface, DHC
 Therefore the container will not be able to reach the host via `ipvlan` interface.
 Be sure to also have container join a network that provides connectivity to the host (e.g. `ptp`).
 * A single master interface can not be enslaved by both `macvlan` and `ipvlan`.
+
+### Using ipvlan and dhcp
+There are two considerations when using "ipvlan" plugin together with "dhcp" IPAM plugin.
+
+First, since ipvlan driver uses the destination address in the IP packet when deciding to which slave the packet should be forwarded to, it will not work if the container interface is not yet assigned an IP.
```

Comment:
First, since the kernel ipvlan driver uses the IP packet's destination address when choosing the slave interface to which it should be forwarded, the driver cannot perform this forwarding if the container interface has not yet been assigned an IP address.